lib/move/__tests__/move.test.js

```diff
@@ -12,14 +12,12 @@ const { differentDevice, ifCrossDeviceEnabled } = require('./cross-device-utils'
 const describeIfWindows = process.platform === 'win32' ? describe : describe.skip
 
 function createAsyncErrFn (errCode) {
-  const fn = function (...args) {
+  async function fn () {
     fn.callCount++
-    const callback = args.pop()
-    setTimeout(() => {
-      const err = new Error()
-      err.code = errCode
-      callback(err)
-    }, 10)
+    const err = new Error()
+    err.code = errCode
+
+    return Promise.reject(err)
   }
```

Comment:
The `move` now uses the Promise version of the `fs.rename` API (instead of the callback version), so the stub here needs to be changed.